packages/jest-runtime/src/__tests__/runtime_require_resolve.test.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import type {Config} from '@jest/types';
+import type Runtime from '..';
```

Comment:
```suggestion
import type {RuntimeType as Runtime} from '..';
```

at least that's how it's imported in other places

Reply:
Seems to not matter for this case though (context https://github.com/facebook/jest/pull/9717), and it would give us one more place to clean up when we ship ESM, so I'd prefer to leave it :upside_down_face: